package.json

```diff
@@ -71,4 +67,4 @@
     "url": "https://github.com/electron/docs-parser/issues"
   },
   "homepage": "https://github.com/electron/docs-parser#readme"
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Good call - VS Code insisted on that change for some reason. 😅